### ISSUE TITLE
opening sentence

### ISSUE THREAD

**Issue by paulamlb** [DemocracyEarth/paper#149]
Hey all,

It seems to me that the first sentence is not very _musical_, for a lack of a better word. Is it just me? I think the first sentence should be killer. I've been wrestling on how to make it nicer but perhaps someone else can do this better than me. @medied you are usually good at this! Suggestions?

"In a world that has succeeded in the globalization of financial assets while keeping political rights enclosed to territories, we need to build new models of democratic governance that enable humanity to collaborate and address pressing global issues."

**Comment by SFSandra** [comment]
I agree, the first sentence sets the tone for everything that comes after.

**Comment by santisiri** [comment]
> A spectre is haunting Europe — the spectre of communism.

**Comment by medied** [comment]
Hmm agree the first sentence could be a bit better. Nothing really wrong with it per se, seems it's just a bit too long or heavy. I understand why _musical_ comes to mind. 

Gotta think about it 🤔

**Comment by medied** [comment]
_While the globalization of financial assets are a reality but political rights are enclosed to territories, we need to build new models of democratic governance that enable humanity to collaborate and address pressing global issues. Democracy Earth Foundation is building free, open source software for incorruptible blockchain-based voting within institutions..._

Not digging starting with 'while' though.

_New models of democratic governance that enable humanity to collaborate and address pressing global issues are needed as long as the globalization of financial assets are a reality but political rights are enclosed to territories. Democracy Earth Foundation is building free, open source software for incorruptible blockchain-based voting within institutions..._

Maybe better.

_The world has witnessed the globalization of financial assets while political rights are still enclosed to territories. We need to build new models of democratic governance that enable humanity to collaborate and address pressing global issues. Democracy Earth Foundation is building free, open source software for incorruptible blockchain-based voting within institutions..._

Another option is separating into two sentences. 

Or coming up with something different altogether, a la Marx like Santi hints.

Opening sentences can be hard to nail sometimes 😐

**Comment by santisiri** [comment]
I don't think this is an issue at all to be honest.

**Comment by SFSandra** [comment]
I like 2 sentence option best, I like starting with "The world" - great immediate global contextualizer the paper, for the reader, for us. 
"The world has witnessed the globalization of financial assets while political rights are still enclosed to territories. We need to build new models of democratic governance that enable humanity to collaborate and address pressing global issues. Democracy Earth Foundation is building free, open source software for incorruptible blockchain-based voting within institutions..."

_Every writer knows the last thing you know is what to put first._

**Comment by paulamlb** [comment]
The issue with two sentences is that in the very first sentence we must say what we are here for... i know it's a lot of info for one sentence but all very necessary info. So I guess the best solution is Eduardo's which opens much stronger then the one we have currently, and doesn't feel as broken:

New models of democratic governance that enable humanity to collaborate and address pressing global issues are needed as long as the globalization of financial assets are a reality but political rights are enclosed to territories.

**Comment by medied** [comment]
Ha @santisiri is like 'hi can you not see me standing right here?'

Yea I'm fine either way. Maybe regardless of leaving it as is or putting something else, if the opening sentence is truly weak, it's something that will be signaled from community and we could readjust then
